src/entities/user.ts

```diff
@@ -12,9 +12,18 @@ import { Project, ProjStatus } from './project';
 import { Donation } from './donation';
 import { Reaction } from './reaction';
 import { AccountVerification } from './accountVerification';
-import { ProjectStatusReason } from './projectStatusReason';
 import { ProjectStatusHistory } from './projectStatusHistory';
 
+export const publicSelectionFields = [
+  'user.id',
+  'user.walletAddress',
+  'user.name',
+  'user.firstName',
+  'user.lastName',
+  'user.url',
+  'user.avatar',
+];
```

Comment:
We can keep it as it is, but the user entity always must be mapped to `user` name in the query.